ui/src/app/pages/workspace/workspace-edit.spec.tsx

```diff
@@ -156,64 +159,51 @@ describe('WorkspaceEdit', () => {
     mockEnsureBillingScope = jest.spyOn(Authentication, 'ensureBillingScope');
     mockHasBillingScope.mockImplementation(() => false);
     mockEnsureBillingScope.mockImplementation(() => {});
+    // Since we are testing delays in these tests, it is helpful to set no dleay in user actions
+    // themselves, so we can focus on delays in our apis:
+    // https://github.com/testing-library/user-event/issues/833
+    user = userEvent.setup({ delay: null });
```

Comment:
Since I [removed the tests that involved delays](https://github.com/all-of-us/workbench/pull/8511/files#r1581530261), this should no longer be necessary.